### PR TITLE
added volume mounts to extractors

### LINKED ISSUE #8: add volumes for extractors
Add volume and volume mounts for clowder extractors

### PR DESCRIPTION
**Pull request by minump**
Added volume mounts to extractors using extraVolumes and extraVolumeMounts options in helm chart

**Fixes https://github.com/clowder-framework/clowder-helm/issues/8**

### CODE REVIEW

**Comment by minump** [comment]
@robkooper ready for review.

File: values.yaml [hunk 1]
```diff
@@ -136,6 +136,14 @@ extractors:
     # env:
     #   - name: extractor.bump.sh
     #     value: "bar"
+    extraVolumes:
```
Comment (robkooper) [inline values.yaml:139]: can you add `[]` to this, so it knows it needs to be an arrray.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## Unreleased
```
Comment (robkooper) [inline CHANGELOG.md:3]: bump patch version here and in `chart.yaml`

**Review by robkooper — CHANGES_REQUESTED** [review]
small tweaks so we can release.

File: values.yaml (hunk 1 above)
Comment (minump) [reply values.yaml]: updated.

File: CHANGELOG.md (hunk 2 above)
Comment (minump) [reply CHANGELOG.md]: updated to 0.16.7

**Comment by minump** [comment]
Ready for review @robkooper

**Review by robkooper — APPROVED** [review]

**Pull request merged**